get_all_manifests.sh

```diff
@@ -29,6 +29,7 @@ declare -A ODH_COMPONENT_MANIFESTS=(
     ["llamastackoperator"]="opendatahub-io:llama-stack-k8s-operator:odh@226e911cca9bf7efa1e632860613087b0bf14d74:config"
     ["trainer"]="opendatahub-io:trainer:main@8ddd0ab935ee68bdc05a0adf843720586617d7b0:manifests"
     ["maas"]="opendatahub-io:maas-billing:main@5e8996752cb625af3dcfb3d16520f137aee7b83c:deployment"
+    ["mlflowoperator"]="opendatahub-io:mlflow-operator:main:config"
```

Comment:
For the sha upgrade tool to work correctly, it is needed to set a commit sha. This is the actual last commit sha. Once merged, the GHA will update it each morning

```suggestion
    ["mlflowoperator"]="opendatahub-io:mlflow-operator:main@dbaa99423f08a170ba5b439d07a4b47e0699bf29:config"
```